templates/launch.html

```diff
@@ -102,7 +102,16 @@
     </div>
 
     <div class="field-container">
-        <input type="submit" value="Open Survey" class="qa-btn-submit-dev btn"/>
+        <label for="roles">Roles</label>
+        <select id="roles" name="roles" multiple="multiple" class="qa-roles">
+            <option name="flusher" value="flusher">flusher</option>
+            <option name="dumper" value="dumper">dumper</option>
+        </select>
+    </div>
+
+    <div class="field-container">
+        <input type="submit" name="action" value="Open Survey" class="qa-btn-submit-dev btn"/>
```

Comment:
maybe these should be more `machine` friendly now e.g. `open_survey` and `flush_data` ?